src/lightning/pytorch/strategies/__init__.py

```diff
@@ -12,17 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from lightning.fabric.strategies.registry import _StrategyRegistry
-from lightning.pytorch.strategies.ddp import DDPStrategy  # noqa: F401
-from lightning.pytorch.strategies.deepspeed import DeepSpeedStrategy  # noqa: F401
-from lightning.pytorch.strategies.fsdp import FSDPStrategy  # noqa: F401
+from lightning.pytorch.strategies.ddp import DDPStrategy
+from lightning.pytorch.strategies.deepspeed import DeepSpeedStrategy
+from lightning.pytorch.strategies.fsdp import FSDPStrategy
 from lightning.pytorch.strategies.ipu import IPUStrategy  # noqa: F401
-from lightning.pytorch.strategies.parallel import ParallelStrategy  # noqa: F401
-from lightning.pytorch.strategies.single_device import SingleDeviceStrategy  # noqa: F401
+from lightning.pytorch.strategies.parallel import ParallelStrategy
+from lightning.pytorch.strategies.single_device import SingleDeviceStrategy
 from lightning.pytorch.strategies.single_tpu import SingleTPUStrategy  # noqa: F401
-from lightning.pytorch.strategies.strategy import Strategy  # noqa: F401
+from lightning.pytorch.strategies.strategy import Strategy
 from lightning.pytorch.strategies.utils import _call_register_strategies
 from lightning.pytorch.strategies.xla import XLAStrategy  # noqa: F401
 
 _STRATEGIES_BASE_MODULE = "lightning.pytorch.strategies"
 StrategyRegistry = _StrategyRegistry()
 _call_register_strategies(StrategyRegistry, _STRATEGIES_BASE_MODULE)
+
+__all__ = [
+    "DDPStrategy",
+    "DeepSpeedStrategy",
+    "FSDPStrategy",
+    "ParallelStrategy",
+    "SingleDeviceStrategy",
+    "Strategy",
+]
```

Comment:
IPU and XLA are missing from this list

Reply:
yes as they are not expected to be mainstream

Reply:
This would be a breaking change

Reply:
it would not as it is not there yet, this is addition, not removal